test/mp4-emsg.test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var QUnit = require('qunit'),
+  emsg = require('../lib/mp4/emsg'),
+  generateEmsgBoxData = require('./utils/mp4-helpers').generateEmsgBoxData,
+  messageData = new Uint8Array([0x64, 0x61, 0x74, 0x61]); // data;
+
+QUnit.module('EMSG Parsing');
+
+QUnit.test('Can parse a v0 emsg box', function(assert) {
+  var boxData = generateEmsgBoxData(0, messageData);
+  var parsedBox = emsg.parseEmsgBox(boxData);
+
+  assert.equal(parsedBox.scheme_id_uri, 'urn:foo:bar:2023\0', 'v0 box has expected scheme_id_uri');
+  assert.equal(parsedBox.value, 'foo.bar.value\0', 'v0 box has expected value');
+  assert.equal(parsedBox.timescale, 100, 'v0 box has expected timescale');
+  assert.equal(parsedBox.presentation_time, undefined, 'v0 box has expected presentation_time');
+  assert.equal(parsedBox.presentation_time_delta, 1000, 'v0 box has expected presentation_time_delta');
```

Comment:
No change required but out of curiosity, why is the presentation_time value undefined here, and the presentation_time_delta is undefined in the other test?

Reply:
`presentation_time` and `presentation_time_delta` usage is dependent on the emsg box version. `v0` boxes use `presentation_time_delta` and `v1` boxes use `presentation_time`. I'm not exactly sure as to _why_ they are that way, but it does change the calculation for the cueTime, or the `ST` from the [spec](https://dashif-documents.azurewebsites.net/Events/master/event.html#emsg-format).